packages/shared/lib/clients/orchestrator.ts

```diff
@@ -822,27 +826,46 @@ export class Orchestrator {
                 recordsService: props.recordsService,
                 initiator: props.initiator
             });
-        }
+        };
```

Comment:
I would have like the commands to both run via temporal and via the orchestrator so that the state of a schedule is kept in-sync  but unfortunately the RUN_FULL command has a bunch of side effect (deleted records, resetting lastSyncDate, ...) so I am only running both when not a RUN_FULL.

Reply:
that's a good point. we would need to pass the sync type as part of the task args but that's feasible

Reply:
I am not gonna do it as past of this PR though